scripts/integration-tests/e2e-prettier.sh

```diff
@@ -27,6 +27,8 @@ if [ "$BABEL_8_BREAKING" = true ] ; then
   sed -i 's/const generate = babelGenerator.default/const generate = babelGenerator/' scripts/build/transform/index.js
   sed -i 's/,"updateContext":null//g' tests/integration/__tests__/__snapshots__/debug-print-ast.js.snap
   rm tests/unit/__snapshots__/visitor-keys.js.snap
+  # Update recordAndTuple usage
+  sed -i 's/\["recordAndTuple", { syntaxType: "hash" }\]/"recordAndTuple"/' src/language-js/parse/babel.js
```

Comment:
/cc @fisker We plan to remove the `syntaxType` option in Babel 8. The syntax will always be hash.